azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -418,8 +425,13 @@ private void createPod(final int executionId) throws ExecutorManagerException {
     if (flowParam != null && !flowParam.isEmpty()) {
       logger.info("ExecId: {}, Flow Parameters are: {}", executionId, flowParam);
     }
-
-    final VersionSet versionSet = fetchVersionSet(executionId, flowParam, jobTypes);
+    // Create all image types by adding azkaban base image, azkaban config and all job types for
+    // the flow.
+    final Set<String> allImageTypes = new TreeSet<>();
+    allImageTypes.add(AZKABAN_BASE_IMAGE);
+    allImageTypes.add(AZKABAN_CONFIG);
+    allImageTypes.addAll(jobTypes);
+    final VersionSet versionSet = fetchVersionSet(executionId, flowParam, allImageTypes);
```

Comment:
Flow parameter for base and config will work fine right?

Reply:
Yes flow parameter override for azkaban base and config should work.